src/components/wallet/earn/Validate/RegisterNode.vue

```diff
@@ -65,8 +65,9 @@
                 </h4>
             </div>
             <div class="requirement_title">
-                <fa v-if="isNodeRegistered" class="success_status_icon" icon="check-circle"></fa>
-                <fa v-else class="error_status_icon" icon="times-circle"></fa>
+                <fa class="info_status_icon" icon="info-circle"></fa>
+                <!-- <fa v-if="isNodeRegistered" class="success_status_icon" icon="check-circle"></fa>
```

Comment:
Remove commented code if it's not needed

Reply:
@Ayagoumi Deleted the commented code